lib/travis/tools/github.rb

```diff
@@ -278,11 +278,22 @@ def security(type, key, arg, name)
         end
 
         def file(path, default = nil)
-          path        &&= File.expand_path(path)
+          gpg_path    = File.expand_path("#{path}.gpg")
```

Comment:
This implementation checks for gpg-encrypted files on all `#file` calls, which is far more than just the credentials. I don't know if that is a good idea.

Reply:
Indeed, thanks for reviewing (and sorry for the delay!). I've changed the code to avoid this.